portfolio/src/main/webapp/style.css

```diff
@@ -88,8 +100,8 @@ body {
 
 .main {
   margin: auto auto;
-  width: 650px;
-  top: 120px;
+  width: 80ch;
```

Comment:
width is specified as 100% below, you can remove this width.

Reply:
If I change this width it changes the appearance.